lib/structuraid_core/engineering/analysis/footing/centric_combined_two_columns.rb

```diff
@@ -0,0 +1,85 @@
+module StructuraidCore
+  module Engineering
+    module Analysis
+      module Footing
+        class CentricCombinedTwoColumns < Base
+          include Utils::Data
+          include Utils::Centroid
+          include Utils::ShearMoment
+
+          def initialize(footing:, loads_from_columns:, section_direction:)
+            if ORTHOGONALITIES.none?(section_direction)
+              raise Engineering::Analysis::SectionDirectionError.new(section_direction, ORTHOGONALITIES)
+            end
+
+            @footing = footing
+            @loads_from_columns = loads_from_columns
+            @section_direction = section_direction
+          end
+
+          def build_geometry(coordinates_system)
+            @coordinates_system = coordinates_system
+
+            coordinates_system.relative_locations.clear
```

Comment:
I think it would be a better interface to have a `CoordinatesSystem#clear_locations` method to remove locations, instead of doing this nested call. What do you think?

Reply:
It is a thing like the Law Of Demeter, right?